proj-sys/build.rs

```diff
@@ -3,13 +3,67 @@ use cmake;
 use flate2::read::GzDecoder;
 use std::fs::File;
 
+#[cfg(target_env = "msvc")]
+use vcpkg;
+#[cfg(target_env = "msvc")]
+use build;
 use pkg_config;
 use std::env;
 use std::path::PathBuf;
 use tar::Archive;
 
 const MINIMUM_PROJ_VERSION: &str = "7.2.1";
 
+#[cfg(target_env = "msvc")]
+fn try_vcpkg() -> Result<std::path::PathBuf, Box<dyn std::error::Error>> {
+    build::link("shell32", true);
+    build::link("ole32", true);
+    let lib = vcpkg::Config::new()
+    .emit_includes(true)
+    .find_package("proj");
```

Comment:
We need to constrain the version based on `MINIMUM_PROJ_VERSION` like we do with pkg_config. Is that possible with vcpkg?

Reply:
I look you to check the version

Reply:
I can't get the library version with vcpkg.
I can walk to library parents directory to read json file of the package if really needed

Reply:
Sorry for the incredibly long delay (almost a year!). 

The nightmare scenario is that someone has an old unsupported version of libproj and this library then gives them weird results because of it. On non-windows platforms we can enforce that this doesn't happen.

I'm not sure how likely this is in practice. Anybody else want to weigh in?